distro/configs/openfn/openfn.env

```diff
@@ -0,0 +1,192 @@
+# Default values are optimized for production to avoid having to configure
+# much in production.
+#
+# However it should be easy to get going in development too. If you see an
+# uncommented option that means it's either mandatory to set or it's being
+# overwritten in development to make your life easier.
+
+# Set this up to handle GitHub App configuration
+# GITHUB_APP_ID=12345
+# GITHUB_CERT=Base64-encoded-private-key
+
+# Set this up to handle SalesForce OAuth credentials
+# SALESFORCE_CLIENT_ID=3MVG9_ghE
+# SALESFORCE_CLIENT_SECRET=703777B
+
+# Set this up to handle Google OAuth credentials (ex: GoogleSheets)
+# GOOGLE_CLIENT_ID=660274980707
+# GOOGLE_CLIENT_SECRET=GOCSPX-ua
+
+# Choose an admin email address and configure a mailer. If you don't specify
+# mailer details the local test adaptor will be used and mail previews can be
+# viewed at localhost:4000/dev/mailbox
+EMAIL_ADMIN='admin@openfn.org'
+# MAILGUN_API_KEY='some-key'
+# MAILGUN_DOMAIN='some-domain'
+
+# You should generate a random string of 64+ characters for this value in prod.
+# You can generate a secure secret by running: ./run secret
+SECRET_KEY_BASE=please_generate_a_more_secure_unique_secret_value_for_your_project
+
+# Which environment is running? MIX_ENV should be "dev" or "prod" and NODE_ENV
+# should be "production" or "development". When MIX_ENV is set to prod you'll
+# automatically be set to build and run releases instead of using mix.
+#MIX_ENV=prod
+#NODE_ENV=production
+MIX_ENV=prod
+NODE_ENV=production
+
+# Override the default log level
+# Must be a valid level, see: https://hexdocs.pm/logger/1.12.3/Logger.html#module-levels
+#LOG_LEVEL=debug
+
+# The URL that will be generated through out your app. When you combine all 3
+# values it should be the URL that visitors access in their browser / client.
+#URL_SCHEME=https
+#URL_HOST=
+#URL_PORT=443
+URL_SCHEME=http
+URL_HOST=localhost
+URL_PORT=4000
```

Comment:
@taylordowns2000 should this be changed in production?

If i my domain is `msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com`, should the env on the server look like below?
```
URL_SCHEME=https
URL_HOST=msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com
ORIGINS=//localhost:4000, //msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com:4000
```

Also if `msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com:4000` is mapped to `lime-dev-openfn.ocg.msf.org` using DNS, should the env on the server look like below?
```
URL_SCHEME=https
URL_HOST=msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com
ORIGINS=//localhost:4000, //msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com:4000, lime-dev-openfn.ocg.msf.org
```

cc @pirupius @pirupius

Reply:
yes it should. that looks great.

Reply:
@taylordowns2000 it seams that the ORIGINS env variable only works when only one domain is specified.  

> ```
> ORIGINS=//localhost:4000, //msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com:4000, lime-dev-openfn.ocg.msf.org
> ```
this failed 

But `ORIGINS=//msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com:4000` worked

Reply:
Thats how we solved infinite loading of the set-up page